app/src/main/java/com/trantan/music53/ui/search/SearchActivity.java

```diff
@@ -0,0 +1,22 @@
+package com.trantan.music53.ui.search;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+
+import com.trantan.music53.R;
+
+public class SearchActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_search);
+    }
+
+    public static Intent getSearchIntent(Context context) {
```

Comment:
getIntent là được rùi em
```suggestion
    public static Intent getIntent(Context context) {
```